config/application.rb

```diff
@@ -16,6 +16,9 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
     config.time_zone = 'Eastern Time (US & Canada)'
+    config.autoload_paths << Rails.root.join('lib')
```

Comment:
Why these? I think it's fine, but just curious if it matters for this PR in some way?

Reply:
I'm embarrassed to say I have no idea.